fluent-gecko/makefile

```diff
@@ -2,9 +2,9 @@ PACKAGE := fluent-gecko
 
 include ../common.mk
 
-build: MessageContext.jsm Localization.jsm DOMLocalization.jsm l10n.js
+build: FluentBundle.jsm Localization.jsm DOMLocalization.jsm l10n.js
 
-MessageContext.jsm: $(SOURCES)
+FluentBundle.jsm: $(SOURCES)
```

Comment:
@zbraniecki What's your preference wrt. renaming this JSM? Should we do it?

Reply:
No preference. Happy to follow your preference.

Reply:
No preference. Open to renaming it. At least to Fluent.jsm :)